tests/test_checkpointing.py

```diff
@@ -165,6 +167,15 @@ def __init__(self, requires_grad):
             MockModel.return_value.named_parameters.return_value = [("model1", MockParams(True))]
             MockModel.use_bert = 1
             model = MockModel()
+            test_trainer, train_params, opt_params, schd_params = trainer.build_trainer(
+                self.args,
+                ["wic"],  # here, we use WIC twice to reduce the amount of boiler-plate code
+                model,
+                self.args.run_dir,
+                self.wic.val_metric_decreases,
+                phase="pretrain",
+            )
+
             test_trainer, train_params, opt_params, schd_params = trainer.build_trainer(
```

Comment:
You initialize these variable names twice. Disambiguate or use _ if you really don't need the first set.